_i18n/en/documentation/general/gpodder.md

```diff
@@ -0,0 +1,43 @@
+<!-- mdpo-disable -->
+
+{% capture img-devices %}
+{% include image.html
+   alt="
+       <!-- mdpo-enable-next-line -->
+       devices
+
+       "
+   loc="/assets/images/documentation"
+   file="gpodder-devices.png"
+%}
```

Comment:
Hmm this is indeed quite a bit of text. Would it be possible to only include the URL, not the full image? Or does that add whitespaces? Then we do not need the mdpo enable/disable code.

Something like:

```
![devices]({% include localized-documentation-image file="gpodder-devices.png" %})
```

Reply:
Downside is that there's a bigger chance that translators mess up, as they see the whole code rather than only the translatable text. Not sure whether risk of translators making a mistake is smaller/bigger than site code contributors. So I can change in line with your suggestion :)

Reply:
It works but the URLs must be captured first, otherwise the images don't work due to spaces:
```
{% capture GP-url %}{% include image-url.liquid folder='badges' file='get-it-on-google-play.png' %}{% endcapture %}
[<img alt="Get it on Google Play" src="{{ GP-url | strip }}" height="80px"/>](https://play.google.com/store/apps/details?id=de.danoeh.antennapod)
![get on GP]({{ GP-url | strip }})
```

| |full image|url only|
|---|---|---|
|capture|only when code otherwise becomes hard to read and in lists|always|
|translators|deal only with text to translate|see & have to copy md tag (html tag on download page)|
|code complexity|more (particularly in md files)|less|

I'm inclined so use both includes: one for more full html image tags with class & size, one for url-only for md image tags.

Reply:
Hmm, having two different includes could be more confusing than helpful for contributors. I would say let's just keep it the way it is now :)